### PR TITLE
Update the FAQ section of scholarx 2023 page

### LINKED ISSUE #1513: Update the dates FAQ section in scholarx 2023 page
**Is your feature request related to a problem? Please describe.**
Currently, the dates in the FAQ section of scholarx 2023 page are outdated. 

**Describe the solution you'd like**
Update the dates according to the schedule for scholarx 2023.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose

The purpose of this PR is to fix #1513

## Goals

Currently, the dates in the FAQ section of scholarx 2023 page are outdated.

## Approach

Update the dates according to the schedule for scholarx 2023.

### Screenshots

Screenshot 2023-02-20 at 08 16 23
Screenshot 2023-02-20 at 08 16 46

  
### Preview Link

https://pr-1514-sef-site.surge.sh/scholarx/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2023/index.html [hunk 1]
```diff
@@ -418,10 +420,10 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
                   <div id="collapseEight" class="collapse" aria-labelledby="headingEight"
                     data-parent="#accordionExampleTwo">
                     <div class="card-body opacity-8 text-justify">
-                      Applications for this year’s instalment of the
+                      Mentee applications for this year’s instalment of the
                       ScholarX program
-                      open on 31 January 2022 and will close 7 April 2022.
-                      Keep an eye on the ScholarX page on our website for
+                      open on the 1st of April 2023 and will close on the 30st of April 2023.
```
Comment (janithlahirukariyawasam) [inline scholarx/2023/index.html]:
```suggestion
                      open on the 1st of April 2023 and will close on the 30th of April 2023.
```

Comment (Madhawa97) [reply scholarx/2023/index.html]: Thank you @janithlahirukariyawasam

**Comment by anjula-sack** [comment]
Shall we move the 2022 page to the archive as well? @Madhawa97

**Comment by Madhawa97** [comment]
You mean in to the past programs, right? Sure. I'll do @anjula-sack

**Comment by anjula-sack** [comment]
> You mean in to the past programs, right? Sure. I'll do @anjula-sack

sorry yes.

**Review by anjula-sack — APPROVED** [review]
Thank you! @Madhawa97

**Comment by Madhawa97** [comment]
You're welcome @anjula-sack

**Pull request merged**